docs: add cad workflow to README (#117)
openshift/configuration-anomaly-detection: README.md

## Workflow

> 1. PagerDuty webhook receives CHGM alert.

Reviewer: ```suggestion
1. PagerDuty webhook receives CHGM alert from deadmanssnitch
```

> 2. CAD Tekton pipeline is triggered.

Reviewer: ```suggestion
2. CAD Tekton pipeline is triggered via PagerDuty sending a webhook to our Tekton EventListener.
```

3. Logs into AWS account of cluster and checks for stopped/terminated instances.

>     - If unable to access AWS account, sends appropriate service log.

Reviewer: ```suggestion
    - If unable to access AWS account, sends "cluster credentials are missing" service log.
```

4. If stopped/terminated instances are found, pulls AWS CloudTrail events for those instances.
5. If the user of the event is:

>     - Authorized (SRE or OSD managed), escalates the alert to SRE for futher investigation.

Reviewer: Would it make sense to provide a list of authorized roles here?
